src/FakeItEasy/ExceptionMessages.tt

```diff
@@ -0,0 +1,122 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>

```

Comment:
Well, basically, normally a T4 template produces only one output file. It might be possible to produce several files with the same template, but I don't know how. I can try to find out.